docs_user/modules/proc_stopping-infrastructure-management-and-compute-services.adoc

```diff
@@ -2,23 +2,21 @@
 
 = Stopping infrastructure management and Compute services
 
-The source cloud's control plane can be decomissioned,
-which is taking down only cloud controllers, database and messaging nodes.
-Nodes that must remain functional are those running the `Compute`, `Storage`,
-or `Networker` roles (in terms of composable roles covered by
-{OpenStackPreviousInstaller} Heat Templates).
+You must decomission cloud Controller nodes, database nodes, and messaging nodes on the source cloud's control plane. Nodes that are running the Compute, Storage, or Networker roles must remain functional (in terms of composable roles covered by {OpenStackPreviousInstaller} Heat Templates).
```

Comment:
I'm not really sure how the first paragraph in the introduction fits with the second one. What is the user doing in this procedure? Are users decommissioning nodes, or removing packages? Or is removing conflicting packages the same thing as decommissioning the nodes?

Reply:
decomissioning also requires re-provisioning the hardware. Thise guide only covers stopping the services being adopted

Reply:
Ok I think I would revise this paragraph to:

"You must stop cloud Controller nodes, database nodes, and messaging nodes on the source cloud's control plane. However, nodes that are running the Compute, Storage, or Networker roles must remain functional on the control plane."

I removed the "in terms of..." sentence because I'm not convinced it's needed. If we think our customers would be familiar with the roles in tripleo, I would probably exclude it.